app/src/main/java/org/dhis2/utils/resources/FilterResources.kt

```diff
@@ -0,0 +1,101 @@
+package org.dhis2.utils.resources
+
+import android.content.Context
+import org.dhis2.Bindings.toUiText
+import org.dhis2.R
+import org.dhis2.utils.filters.workingLists.RelativePeriodToStringMapper
+import org.hisp.dhis.android.core.common.DateFilterPeriod
+import org.hisp.dhis.android.core.enrollment.EnrollmentStatus
+import org.hisp.dhis.android.core.event.EventStatus
+
+class FilterResources(val context: Context) {
+    fun defaultWorkingListLabel(): String = context.getString(R.string.working_list_default_label)
+    fun todayLabel(): String = context.getString(R.string.filter_period_today)
+    fun yesterdayLabel(): String = context.getString(R.string.filter_period_yesterday)
+    fun lastNDays(days: Int): String =
+        context.getString(R.string.filter_period_last_n_days).format(days)
+
+    fun thisMonthLabel(): String = context.getString(R.string.filter_period_this_month)
+    fun lastMonthLabel(): String = context.getString(R.string.filter_period_last_month)
+    fun thisBiMonth(): String = "This bimonth"
+    fun lastBiMonth(): String = "Last bimonth"
+    fun thisQuarter(): String = "This quarter"
+    fun lastQuarter(): String = "Last quarter"
+    fun thisSixMonth(): String = "This six months"
+    fun lastSixMonth(): String = "Last six months"
```

Comment:
Is there a reason for not using strings from the string.xml file?

Reply:
My bad. Fixing it now